pcbdraw/plot.py

```diff
@@ -135,15 +136,19 @@ def flip(self) -> None:
 def matrix(data: List[List[Numeric]]) -> Matrix:
     return np.array(data, dtype=np.float32)
 
-def pseudo_distance(a: Point, b: Point) -> Numeric:
-    return (a[0] - b[0])**2 + (a[1] - b[1])**2
+# def distance(a: Point, b: Point) -> Numeric:
+#     return math.sqrt((a[0] - b[0])**2 + (a[1] - b[1])**2)
```

Comment:
What's the point of commenting out this function? At the moment, it is not used. However, it makes sense to have it ready to complement `pseudo_distance`. And if we want to get rid of it, we should delete it.

Reply:
I commented it because it isn't used, you can uncomment it when needed. Or just as a reference of what a real distance is.

I usually uncomment unused functions to avoid confusion (people might think this is used), misleading coverage results (code not covered just because is dead), avoid wasting resources, etc.